frontend/src/App.js

```diff
@@ -35,6 +36,12 @@ function App() {
         })}/>
     };
 
+    const renderWarehouse = () => {
+        return <ProtectedComponent conditions={user} render={(() => {
```

Comment:
Is that enough condition in conditions prop?

Reply:
you're right. it's not enough